components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -23,8 +23,82 @@
 /** The minimum height of any item view. */
 static const CGFloat kMinHeight = 48;
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {.top = 12, .right = 16, .bottom = 12, .left = 16};
+
+/// File name of the bundle (without the '.bundle' extension) containing resources.
+static NSString *const kResourceBundleName = @"MaterialTabs";
+
+/// String table name containing localized strings.
+static NSString *const kStringTableName = @"MaterialTabs";
+
+/// Scale factor applied to the title of bottom navigation items when selected.
+const CGFloat kSelectedNavigationTitleScaleFactor = (16.0f / 14.0f);
+
+/// Vertical translation applied to image components bottom navigation items when selected.
+const CGFloat kSelectedNavigationImageYOffset = -2;
+
+@interface MDCTabBarViewItemView ()
+
+@property(nonatomic, strong) UIStackView *contentView;
+@property(nonatomic, strong) UIImageView *iconImageView;
+@property(nonatomic, strong) UILabel *titleLabel;
+
+@end
+
 @implementation MDCTabBarViewItemView
 
+#pragma mark - Init
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    _title = @"";
+
+    self.isAccessibilityElement = YES;
+
+    // Create initial subviews
+    [self initSubviews];
+  }
+  return self;
+}
+
+#pragma mark - UIView
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+
+  CGRect contentBounds = UIEdgeInsetsInsetRect(self.bounds, kEdgeInsets);
+  self.contentView.frame = contentBounds;
+}
+
+#pragma mark - UIAccessibility
+
+#pragma mark - Private
+
+- (void)initSubviews {
+  if (!_contentView) {
+    _contentView = [[UIStackView alloc] initWithFrame:CGRectZero];
```

Comment:
Tip: Please set the autoresizing mask explicitly in these subviews. That way it's easier for maintainers to reason about the generated constraints.

Reply:
This is changed to be a UIView. I will just resolve this comment.